cloudformation/ci_resources.yml

```diff
@@ -169,7 +169,11 @@ Resources:
   # | display eventSource, eventName
   # | dedup eventSource, eventName
   #
-  GrantCloudFormationExecutionAccessPolicyA:
+
+  # there are multiple policies to prevent each one becoming too large
+
+  # iam permissions are restricted to just act on this account so are in a separate policy
+  GrantCloudFormationExecutionAccessIAMPolicy:
```

Comment:
Would it be worth writing exactly what the intended use of the policies are in this file, for quick reference later?

Reply:
its documented at  https://docs.aws.amazon.com/service-authorization/latest/reference/reference_policies_actions-resources-contextkeys.html - a link to this is included in this file as a comment for exactly this reason